kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMDocumentTest.java

```diff
@@ -91,40 +91,40 @@ public void that_FOR_PREVIEW_useSystemMenuBarProperty_is_enabled() throws Except
         FXOMObject fxomObject = classUnderTest.searchWithFxId("theMenuBar");
 
         assertTrue(fxomObject.getSceneGraphObject() instanceof MenuBar);
-        assertTrue("for preview, useSystemMenu is expected to be enabled",
-                ((MenuBar) fxomObject.getSceneGraphObject()).useSystemMenuBarProperty().get());
+        assertTrue(((MenuBar) fxomObject.getSceneGraphObject()).useSystemMenuBarProperty().get(),
+                "for preview, useSystemMenu is expected to be enabled");
 
         String generatedFxml = classUnderTest.getFxmlText(false);
         assertTrue(generatedFxml.contains("useSystemMenuBar=\"true\""));
     }
 
     @Test
     public void that_useSystemMenuBarProperty_is_disabled_on_MacOS() throws Exception {
-        assumeTrue("MacOS", OS.get() == OS.MAC);
+        assumeTrue(OS.get() == OS.MAC, "MacOS");
```

Comment:
Same rules for descriptions applies to JUnit5 assumptions. A description is now always the last method argument.